src/SprykerSdk/Zed/ComposerConstrainer/Business/ComposerConstrainerFacadeInterface.php

```diff
@@ -24,4 +24,11 @@ public function updateConstraints(): ComposerConstraintCollectionTransfer;
      * @return \Generated\Shared\Transfer\ComposerConstraintCollectionTransfer
      */
     public function validateConstraints(): ComposerConstraintCollectionTransfer;
+
+    /**
+     * @api
+     *
+     * @return \Generated\Shared\Transfer\ComposerConstraintCollectionTransfer
+     */
+    public function validateCoreAndForeignConstraints(): ComposerConstraintCollectionTransfer;
```

Comment:
update method is missing in the interface

Reply:
Fixed.